src/nni_manager/training_service/common/util.ts

```diff
@@ -40,9 +40,10 @@ import { file } from "../../node_modules/@types/tmp";
  */
 export async function validateCodeDir(codeDir: string) : Promise<number> {
     let fileCount: number | undefined;
-
+    let fileNameValid: boolean = true;
     try {
         fileCount = await countFilesRecursively(codeDir);
+        fileNameValid = await validateFileNameRecursively(codeDir);
     } catch(error) {
         throw new Error(`Call count file error: ${error}`);
```

Comment:
this error message is not proper any more. Suggest to do countFile and validateFile one by one separately.

Reply:
fixed, validate filename and filecount separately.